### PR TITLE
Add overloads for fld and cld

### PR DESCRIPTION
**Pull request by sethaxen**
This PR adds overloads for `fld` and `cld`, which currently throw a `MethodError` on master.

### CODE REVIEW

File: test/DualTest.jl [hunk 1]
```diff
@@ -137,6 +137,14 @@ for N in (0,3), M in (0,4), V in (Int, Float32)
         @test round(FDNUM2) === round(PRIMAL2)
         @test round(NESTED_FDNUM) === round(PRIMAL)
 
+        @test fld(FDNUM, FDNUM2) === fld(PRIMAL, PRIMAL2)
+        @test fld(FDNUM, PRIMAL2) === fld(PRIMAL, PRIMAL2)
+        @test fld(PRIMAL, FDNUM2) === fld(PRIMAL, PRIMAL2)
+
+        @test cld(FDNUM, FDNUM2) === cld(PRIMAL, PRIMAL2)
+        @test cld(FDNUM, PRIMAL2) === cld(PRIMAL, PRIMAL2)
```
Comment (oxinabox) [inline test/DualTest.jl:145]: Did something go weird with the indebting here?

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "ForwardDiff"
 uuid = "f6369f11-7733-5829-9624-2563aa707210"
-version = "0.10.25"
+version = "0.10.26"
```
Comment (oxinabox) [inline Project.toml:3]: I believe @KristofferC prefers to let a few changes to build up before cutting a release?

**Review by oxinabox — APPROVED** [review]
Math checks out

File: Project.toml (hunk 2 above)
Comment (devmotion) [reply Project.toml:3]: There's already an open PR for making a new release (with the NaNMath compat PR) which nobody has reviewed for weeks...

File: test/DualTest.jl (hunk 1 above)
Comment (KristofferC) [reply test/DualTest.jl:145]: I don't really see any weirdness.

Comment (sethaxen) [reply test/DualTest.jl:145]: Yes, @oxinabox, what seems weird?

Comment (oxinabox) [reply test/DualTest.jl:145]: Never mind, probably something off with the rendering on my phone.
It looked like this was at a different indent level to rest of file

**Pull request merged**